src/utils.ts

```diff
@@ -353,6 +372,15 @@ export const getPrDescription = (body: string = '', story: PivotalStory): string
   </table>
 </details>
 <br />
+
+${requireDesignReview && `<details open>
+  <summary><strong>Pivotal Design Review</strong></summary>
+  <br />
+  
+  - [ ] Design review is required for this PR, make sure to ask @AMS-00 for review once finished.
```

Comment:
There's a few things in here that I think might be worth considering.

1. Does Anna really want a notification every time you add a review, and what happens when she hires a new colleague?
2. How might we want to extend this in the future - imagine we had a security team and wanted a security review for instance? Or we wanted Anca and Carly to have a review?
3. How could a non Mention Me user use this feature?

As a clue, you may be better off creating something that looks like this:

```
# Reviews required
- [ ] Design
- [ ] Security
- [ ] Something else
```

Reply:
I really did not want to use the explicit IDs to compare, however couldn't find any API end point to extract the reviewer or review type name based on the Ids. I will dig some more

Reply:
Found it now :)